apps/site/assets/ts/components/Badge.tsx

```diff
@@ -2,14 +2,17 @@ import React from "react";
 
 const Badge = ({
   text,
-  contextText
+  contextText,
+  bgClass
 }: {
   text: string;
   contextText?: string;
+  bgClass?: string;
```

Comment:
is this just to change background color? Assuming any class can be passed here

Reply:
yeah technically any class can be passed. but the way it's used in the `Badge`, it will replace/remove the default background color.